WISE-Community/WISE
# Styled the edit run settings dialog

To test:
- Log in as a teacher.
- Select "Edit Settings" from the run menu for a variety of runs.
- Test that editing all the settings work.
- Test that invalid operations are not allowed and correct error messages appear:
-- Deleting period that already has students
-- Lowering max students per team to 1 when there are already workgroups with >1 student

Closes #1680.
